cafebabel/static/css/styles.css

```diff
@@ -845,41 +845,63 @@ button.follow.active:after {
 .create-article-page .file {
   position: relative;
   width: 100%;
-  height: 30vw;
+  height: auto;
   background: transparent;
   transition: all 0.3s;
 }
 
-.create-article-page canvas {
+@media screen and (min-width: 1200px) {
```

Comment:
Ohh, because of the image width for article that is 1200px I suppose.
Can't we set a max-width/max-height in line 844 instead?
```css
.create-article-page .file canvas,
 .create-article-page .file {
   position: relative;
   width: 100%;
  max-width: 120rem;
  max-height: 46.7rem;
  …
```

Reply:
> We dont want max-height to be 46.7rem on mobile :-(

I'm missing something here. I can't see the `max-height` that is currently applied for mobile and if there's none, I'm not aware of the side effect of a max-height when that height is not reached. Further details will be welcome to me :)